makefiles/dependency_resolution.inc.mk

```diff
@@ -50,4 +50,11 @@ else
     $(shell $(COLOR_ECHO) "$(COLOR_RED)Deprecated modules are in use:$(COLOR_RESET)"\
                           "$(DEPRECATED_MODULES_USED)" 1>&2)
   endif
+  # Warn about used deprecated boards
```

Comment:
I'm not sure this isn't going to cause trouble. does it print when doing, e.g., "BOARD=6lowpan-clicker make info-modules"?

Reply:
True, I guarded it with `ifneq ($(RIOT_CI_BUILD),1)`, which should help against automated checks.  For the developer just looking at it I don't think the warnings are a problem.

Reply:
Yep, same thing is done for BOARDSDIR already this makes sense